packages/newsroom-signup/src/Newsroom.tsx

```diff
@@ -381,17 +404,22 @@ class NewsroomComponent extends React.Component<NewsroomProps & DispatchProp<any
   private getDisabled(section: SECTION): () => boolean {
     // @TODO/tobek Setting everything to enabled for now for testing, but we should work these out.
     const functions = {
-      [SECTION.CONTRACT]: () => {
+      [SECTION.PROFILE]: () => {
         return false;
       },
+      [SECTION.CONTRACT]: () => {
```

Comment:
could include a charter check here, has required fields etc.  I guess that would also require error messaging

Reply:
Because of our disabled checks inside newsroom profile stuff, they can't get to the grant page unless they've completed their charter, so checking they've passed grant step is inclusive of all the stuff before it. We have a bunch of charter logic in there https://github.com/joincivil/Civil/blob/master/packages/newsroom-signup/src/NewsroomProfile/index.tsx#L32-L67 that we can just reuse this way (unless I'm missing something).